src/utils/gwhisperUtils.hpp

```diff
@@ -10,6 +26,15 @@ namespace gwhisper
         /// @return File content as simple string. If file cannot be opened, this function returns the status "FAIL" as as string.
         std::string readFromFile(const std::string f_path);
 
+        /// Utility function for creating a folder at given location under Linux
```

Comment:
does this really only work in linux? (we have macos users too for gWhisper, so we should be compatible)

Reply:
Do we have the possibility to test this function on macos?

Reply:
maybe via GH actions?

Reply:
but if you only use c++ std lib functions I think it should be cross-platform